app/Modules/Admin/Model/EventService.php

```diff
@@ -58,7 +58,7 @@ public function createEvent(ArrayHash $values)
 			'event_series_id' => $values->event_series_id,
 		]);
 
-		$this->addTags($values->tags, $event->id);
+		$this->addTags($values->tags, (int)$event->id);
```

Comment:
Is there a way to check this kind of code style issues in our cool CI tools? @TomasVotruba